File: PythonProblems/TwoSum.py
class Solution(object):#https://leetcode.com/problems/two-sum/
    def twoSum(self, nums, target):
        for index, value in enumerate(nums):
            for index2, value2 in enumerate(nums):
                if index != index2:
                    if value+value2 == target:
                        return [index, index2]

File: CPlusPlusProblems/TwoSum.cpp
class Solution { //https://leetcode.com/problems/two-sum/ NOTE: I'm using the brute force method.
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        int n = nums.size();//Create the integer here so you dont have to call the function for the loops.
        for (int i = 0; i < n - 1; i++)//Last index isn't needed cause if it goes there theres no solution.
        {
            for (int v = i + 1; v < n; v++)
            {
                if (nums[i] + nums[v] == target)
                {
                    return {i, v};
                }
                    
            }
        }
        return {};//No solution found, returning an empty vector(array i think?).
    }
};